ff/serving/provider/online.go

```diff
@@ -164,6 +230,69 @@ func (store *redisOnlineStore) CreateTable(feature, variant string, valueType Va
 	}
 	table := &redisOnlineTable{client: store.client, key: key, valueType: valueType}
 	return table, nil
+
+}
+
+// CASSANDRA CREATE TABLE
+func (store *cassandraOnlineStore) CreateTable(feature, variant string, valueType ValueType) (OnlineStoreTable, error) {
+
+	//Create table Name
+	var tableName string
+	tableName = fmt.Sprintf("%s.table%s", store.keyspace, sn.Custom(feature, "[^a-zA-Z0-9_]"))
+
+	//Create table key
+	var keyName string
+	key := cassandraTableKey{store.keyspace, feature, variant}
+	keyName = sn.Custom(key.String(), "[^a-zA-Z0-9_]")
+
+	// Create Table
+	query := fmt.Sprintf("CREATE TABLE %s (%s text PRIMARY KEY, %s text)", tableName, keyName, string(valueType))
+	err := store.session.Query(query).WithContext(ctx).Exec()
+	if err != nil {
+		return nil, &TableAlreadyExists{feature, variant}
```

Comment:
Only error during table production may not be tablealreadyexists, so returning this assumes too much specificity. Have it return the actual err so it isn't swallowed